observability/readme.md

```diff
@@ -1,132 +1,33 @@
 # Observability
 
-Building observable systems enables development teams at CSE to measure how well the application is behaving.
+Building observable systems enables development teams at CSE to measure how well the application is behaving. Observability serves the following goals:
 
-## Goals
+- Provide holistic view of the _application health_.
+- Help measure _business performance_ for the customer.
+- Measure _operational performance_ of the system.
+- Identify and _diagnose failures_ to get to the problem fast.
 
-1. Provide holistic view on the health of the application.
-2. Help measure business performance for the customer.
-3. Measure operational performance of the system.
-4. Identify and diagnose failures to get to the problem fast.
+## Getting started
```

Comment:
Not quite sure of this header. This is a common practice across all fundamentals?

Reply:
I removed it - I was trying to put under same umbrella pillar and insights. I now moved them to upper level.